apiserver/facades/client/application/deployrepository_test.go

```diff
@@ -620,6 +620,7 @@ func intptr(i int) *int {
 
 type deployRepositorySuite struct {
```

Comment:
We do need additional tests as part of the deployRepositorySuite. Please add a test to ensure that resources are wired up correctly in DeployFromRepository.